sql/src/test/java/io/crate/executor/transport/distributed/DistributingConsumerTest.java

```diff
@@ -108,6 +123,7 @@ public void testDistributingConsumerForwardsFailure() throws Exception {
     private DistributingConsumer createDistributingConsumer(Streamer<?>[] streamers, TransportDistributedResultAction distributedResultAction) {
         return new DistributingConsumer(
                 logger,
+                THREAD_POOL.executor(DistributingDownstreamFactory.RESPONSE_EXECUTOR_NAME),
```

Comment:
For the unit test I think it would be enough to use `MoreExecutors.directExecutor()` or `Executors.newFixedThreadPool()` as it won't need the full blown threadPool